packages/flutter/lib/src/material/progress_indicator.dart

```diff
@@ -31,12 +31,22 @@ abstract class ProgressIndicator extends StatefulWidget {
   /// The [value] argument can be either null (corresponding to an indeterminate
   /// progress indicator) or non-null (corresponding to a determinate progress
   /// indicator). See [value] for details.
+  ///
+  /// The [semanticsEnabled] argument must not be null. It defaults to false. If
+  /// this widget provides information that should be announced by screen reading
+  /// software, it should be set to true. If it is true, at least one of
+  /// [semanticsLabel] or [semanticsValue] must be specified.
   const ProgressIndicator({
     Key key,
     this.value,
     this.backgroundColor,
     this.valueColor,
-  }) : super(key: key);
+    this.semanticsEnabled = false,
+    this.semanticsLabel,
+    this.semanticsValue,
```

Comment:
value is typically used for things like TextFields and sliders, I believe this would only have a label. Removing this means you could infer semanticsEnabled from it's presence.

Reply:
My thought was someone would want to use this to advertise the amount of progress for a non-indeterminate indicator.  We could still have `semanticsLabel` drive things, but leave this.  What do you think?

Reply:
In that case it would still be driven the label

Reply:
Yes - though my expectation (which could be wrong) is that indeterminate loaders are much, much more common due to the nature of web requests. In that case, most labels would be something like "Loading..."

Reply:
I think I'd favor keeping the enabled flag and not overloading label with two jobs here, partly for consistency with below.  I'm still not clear on the value/label thing, but we can discuss offline.

Reply:
Since iOS is the only platform differentiating between label and value we should check how they make the native progress-bar accessible on their platform. Do they provide a seperate label and value? Apple's Accessibility Inspector can help here.